identity-hub-dtos/build.gradle.kts

```diff
@@ -1,11 +1,23 @@
 plugins {
     `java-library`
+    `maven-publish`
 }
 
 val jetBrainsAnnotationsVersion: String by project
 val swagger: String by project
 
 dependencies {
     api("org.jetbrains:annotations:${jetBrainsAnnotationsVersion}")
-    implementation("io.swagger.core.v3:swagger-jaxrs2-jakarta:${swagger}")
+    implementation("io.swagger.core.v3:swagger-jaxrs2-jakarta:${swagger}") {
+        exclude(group = "com.fasterxml.jackson.jaxrs", module = "jackson-jaxrs-json-provider")
+    }
+}
+
+publishing {
```

Comment:
not needed at this point but should plan to include `signing` plugin for publishing to Maven Central.

Also will require tasks for including javadoc, source jars  and signing, can also be done at root level